importer/RoRo.py

```diff
@@ -1,29 +1,117 @@
-from Monument import Monument
+from Monument import Monument, Dataset
 import importer_utils as utils
+import importer as importer
 
-class RoRo(Monument):
 
-    def update_labels(self):
-        name = utils.remove_markup(self.denumire)
-        self.add_label("ro", name)
+class RoRo(Monument):
 
     def set_adm_location(self):
-        return
+        counties = self.data_files["counties"]
+        adm_code = self.judetul_iso
+        county_item = utils.get_item_from_dict_by_key(dict_name=counties,
+                                                      search_term=adm_code,
+                                                      search_in="iso_code")
+        if len(county_item) == 1:
+            self.add_statement("located_adm", county_item[0])
+        else:
+            self.add_to_report("judetul_iso",
+                               self.judetul_iso,
+                               "located_adm")
 
-    def set_no(self):
-        self.add_statement("romanian_monument_id", self.cod)
+    def set_location(self):
+        """
+        Set Location property from art wlinked in localitate.
 
-    def __init__(self, db_row_dict, mapping, data_files, existing):
-        Monument.__init__(self, db_row_dict, mapping, data_files, existing)
-        self.update_labels()
-        # self.exists("sq")
+        Run this after set_adm_location. localitate can
+        contain several links (we take the 1st which seems to
+        be the most granular one) and a mix of administrative
+        types. Compare with admin location so that they're not
+        the same.
+        """
+        if self.has_non_empty_attribute("localitate"):
+            loc_item = None
+            if utils.count_wikilinks(self.localitate) > 0:
+                loc_link = utils.get_wikilinks(self.localitate)[0]
+                loc_item = utils.q_from_wikipedia("ro", loc_link.title)
+                adm_item = self.get_statement_values("located_adm")
+                if loc_item and loc_item != adm_item[0]:
+                    self.add_statement("location", loc_item)
+
+            if not loc_item:
+                self.add_to_report("localitate", self.localitate, "location")
+
+    def set_wlm_id(self):
+        """
+        Set the WLM ID.
+
+        Do _not_ use the Romania LMI code (P1770)
```

Comment:
Could you explain the change from P1770 here? Looks like the other property is already in use on Wikidata so through it it's likely where we would find matching items. If it is a fake identifier than I would assume they all get migrated later on.

Reply:
They don't have any external references and according to https://www.wikidata.org/wiki/Wikidata:Property_proposal/Archive/29#P1770 they were originally created to link to rowp.

Reply:
It is still the one which is de facto used and every P1770 entry should also be in the dataset we are trying to import.

That there is a case for getting rid of the Property  (and migrating them to WLM_ID with a prefix) is something which needs to first be raised on Wikidata.

Reply:
OK. reverting to it.